backend/Squirrel.Core/Squirrel.Core.BLL/Hubs/SquirrelHub.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Microsoft.AspNetCore.SignalR;
+using Squirrel.Core.BLL.Interfaces;
+using Squirrel.Core.Common.DTO.SquirrelHub;
+using Squirrel.Core.DAL.Enums;
+
+namespace Squirrel.Core.BLL.Hubs;
+
+public sealed class SquirrelHub : Hub<ISquirrelHub>
+{
+    public override async Task OnConnectedAsync()
+    {
+        var clientId = Guid.NewGuid().ToString();
+        await Clients.Caller.SetClientId(clientId);
+    }
+
+    // TODO: work with data recived from ConsoleApp (save to Db)
+    public async Task ReceiveData(string clientId, string query, DbEngine dbEngine, QueryResultTableDTO queryResultTableDTO)
+    {
+        Console.WriteLine($"------------------------------------------------------------------");
+        Console.WriteLine($"Successfully get data from user '{clientId}'");
```

Comment:
"Successfully get data..." is not in the past tense)
Maybe "Successfully received data..." or something like that